services/venice-controller/src/main/java/com/linkedin/venice/ingestion/control/RealTimeTopicSwitcher.java

```diff
@@ -51,6 +54,10 @@ public RealTimeTopicSwitcher(
     this.destKafkaBootstrapServers = veniceProperties.getBoolean(ConfigKeys.SSL_TO_KAFKA, false)
         ? veniceProperties.getString(ConfigKeys.SSL_KAFKA_BOOTSTRAP_SERVERS)
         : veniceProperties.getString(ConfigKeys.KAFKA_BOOTSTRAP_SERVERS);
+    int kafkaReplicationFactor = veniceProperties.getInt(KAFKA_REPLICATION_FACTOR);
```

Comment:
Can we set some reasonable default value for `KAFKA_REPLICATION_FACTOR` in `VeniceControllerClusterConfig::initFieldsWithProperties`? 

Looks like a bunch of ITs are failing due to `com.linkedin.venice.exceptions.UndefinedPropertyException: Missing required property 'kafka.replication.factor'.`

Reply:
Will set default for both KAFKA_REPLICATION_FACTOR and KAFKA_REPLICATION_FACTOR_LEGACY_SPELLING to 3.

Reply:
Can we get rid of the legacy spelling? Or did we still not migrate off of it?

Reply:
Let me do a code search and delete it if I don't see any reference.

Reply:
They are still in use, should I just migrate them and update this PR?